### PR TITLE
Adding URLMap Sample

### PR DESCRIPTION
**Pull request by sahanacp-hub**
Including the sample in the url: https://cloud.google.com/load-balancing/docs/url-map

```release-note:none
```

### CODE REVIEW

**Review by slevenick — CHANGES_REQUESTED** [review]
Fails to compile. Please make sure you can run https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers successfully prior to PR submission

**Comment by sahanacp-hub** [comment]
Hi, fixed the errors in terraform.yaml. 
I tried running the commands in  https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers but I did not have permissions.
Thanks!

File: mmv1/products/compute/terraform.yaml [hunk 1]
```diff
@@ -2227,6 +2227,17 @@ overrides: !ruby/object:Overrides::ResourceOverrides
...
+        name: "url_map_sample"
+        primary_resource_type: "google_compute_region_url_map"
+        primary_resource_id: "url_map"
+        vars:
+          health_check: "health-check"
+          backend_service: "backend-service"
+          lb_target_proxy: "lb-target-proxy"
+          url_map: "url-map"
+        skip_docs: true
+        min_version: beta
```
Comment (slevenick) [inline mmv1/products/compute/terraform.yaml:2240]: If this is beta-only you need to use `provider = google-beta` in the config

**Review by slevenick — CHANGES_REQUESTED** [review]

File: mmv1/products/compute/terraform.yaml (hunk 1 above)
Comment (sahanacp-hub) [reply mmv1/products/compute/terraform.yaml:2252]: This is not beta-only. Included skip_test: true in terraform.yaml

Comment (slevenick) [reply mmv1/products/compute/terraform.yaml:2252]: Then we need to remove min_version: beta, that prevents it from being marked as a GA test

**Comment by slevenick** [comment]
Looks like the generate-diffs failed to run. Are you able to compile magic-modules locally to produce a Terraform provider?

**Comment by c2thorn** [comment]
Hi @sahanacp-hub 
Sorry for the timing here, but MM no longer supports CGC samples. I believe the "terraform-on-cgc" internal documentation has just been updated with the new workflow, but essentially you'll be making PR's against https://github.com/terraform-google-modules/terraform-docs-samples directly

**Pull request closed without merging**